publisher-sdk/src/androidTest/java/com/criteo/publisher/adview/AdWebViewClientTest.kt

```diff
@@ -186,6 +189,28 @@ class AdWebViewClientTest {
     verify(logger).log(MraidLogMessage.onErrorDuringMraidFileInject(thrownException))
   }
 
+  @Test
+  fun whenHtmlWithMraidIsLoaded_GivenPageFinished_ShouldHaveMraidStateAsDefault() {
+    val mraidHtml = mraidData.getHtmlWithMraidScriptTag()
+
+    loadHtml(mraidHtml)
+    waitForPageToFinishLoading()
+
+    verifyMraidObject(".getState()") { state ->
```

Comment:
It is kind of feeling weird to see a `".getState()"` like this. Why not having `"mraid"` as default value of the method so that other calls could looks like `"mraid.getState()"`

Reply:
Updated code, so now it is just `"getState()"`. The same way it is used in `MraidInteractor`